### PR TITLE
[SQL] Add instruction to use Spark SQL ML

### PR DESCRIPTION
**Pull request by eddyxu**
Add brief introduction about how to use Spark SQL ML for #136

### CODE REVIEW

File: docs/sqlml.rst [hunk 1]
```diff
@@ -0,0 +1,116 @@
+.. toctree::
+   :maxdepth: 1
+
+ML-enabled SQL
+==============
+
+``Rikai`` extends Spark SQL to offer Machine Learning(**ML**)-enabled analytics over
```
Comment (changhiskhan) [inline docs/sqlml.rst:7]: Machine learning repeated in the same sentence.

File: docs/sqlml.rst [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+machine learning datasets.
+
+    Make your Data Warehouse as Smart as your ML models
+
+Similar capabilities have been provided by products, like `BigQuery ML`_ and `Redshift ML`_.
+Rikai SQL ML is designed to be extensible to any Model Registry, no matter it is ``on-prem`` or ``cloud-native``,
+``open-source`` or ``proprietary``.
+
+As a result, ``Rikai SQL-ML`` can be easily integrated into existing machine learning infrastructure,
+and allow your Data Warehouse as smart as your ML models.
```
Comment (changhiskhan) [inline docs/sqlml.rst:17]: to be*

File: docs/sqlml.rst [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+Before we can use ``Rikai SQL-ML``, we need to configure SparkSession:
+
+.. code-block:: python
+
+    spark = (
+        SparkSession
+        .builder
+        .appName("spark-app")
+        .config("spark.jars.packages", "ai.eto:rikai_2.12:0.0.3")
```
Comment (changhiskhan) [inline docs/sqlml.rst:34]: are we doing 0.0.3?

**Review by changhiskhan — APPROVED** [review]
nits mostly. we need to provide a standalone working example in a notebook as well.

File: docs/sqlml.rst (hunk 3 above)
Comment (eddyxu) [reply docs/sqlml.rst]: ok, i will leave `0.0.2` here for now.

**Pull request merged**